src/main/java/teammates/storage/api/FeedbackResponseCommentsDb.java

```diff
@@ -586,10 +419,10 @@ private FeedbackResponseComment getFeedbackResponseCommentEntity(
         return getCommentsWithoutDeletedEntity(feedbackResponseCommentList);
     }
     
-    private Collection<FeedbackResponseComment> getFeedbackResponseCommentEntitiesForSessionInSection(
+    private List<FeedbackResponseComment> getFeedbackResponseCommentEntitiesForSessionInSection(
             String courseId, String feedbackSessionName, String section) {
 
-        Map<String, FeedbackResponseComment> FeedbackResponseCommentList = new HashMap<String, FeedbackResponseComment>();
+        List<FeedbackResponseComment> FeedbackResponseCommentList = new ArrayList<FeedbackResponseComment>();
```

Comment:
this may have potential bug. there're 2 queries here, and the usage of map is to prevent loading a comment twice